resources/variants/cartesio_0.25.inst.cfg

```diff
@@ -0,0 +1,15 @@
+[general]
+name = 0.25 mm
+version = 2
+definition = cartesio
+
+[metadata]
+author = Cartesio
+type = variant
+
+[values]
+machine_nozzle_size = 0.25
+machine_nozzle_tip_outer_diameter = 1.05
+speed_wall = =round(speed_print / 1.25, 1)
```

Comment:
Since you are using this formula in all 4 nozzle configurations, it should be easy to set these for the entire machine using something along the lines of:

```
"overrides": {
  "speed_wall": {
    "value": "round(speed_print / 1.25, 1)"
  }
}
```

This also applies to the other two speed settings.

If you did not intend to override the default formula for speed calculation, I would recommend just removing them.